server/src/test/java/org/elasticsearch/index/mapper/FieldTypeLookupTests.java

```diff
@@ -32,38 +33,6 @@ public void testEmpty() {
         assertEquals(0, size(lookup.filter(ft -> true)));
     }
 
-    public void testFilter() {
```

Comment:
One consequence of this change is that MappingLookup.filter() no longer includes runtime fields.  However, since this is only used in a) tests or b) to get fields to load global ordinals for eagerly, I don't think this is a problem (especially as runtime fields can't be loaded eagerly because they don't have any docvalues anyway).  I think we should probably remove `filter()` entirely in a separate issue and replace it with a `getGlobalOrdinalsFields` method instead.

Reply:
That sounds like going back to what we used to have :D I agree though I think either filter returns runtime fields too, or we have a specific method that does what it needs to do more specifically.